cax/main.py

```diff
@@ -17,6 +17,8 @@ def main():
                         help="Load a custom .json config file into cax")
     parser.add_argument('--log',  dest='log', type=str, default='info',
                         help="Logging level e.g. debug")
+    parser.add_argument('--database',  dest='database_log', type=str, default=True,
```

Comment:
Can't you set the type to `bool`?  I'd rename the variable.  In the DAQ, we use this:  ` parser.add_argument('--secret_mode', action='store_true',
                        help="Never write anything to the runs db. Use only for testing!")`